bigbluebutton-html5/imports/ui/services/auth/index.js

```diff
@@ -217,6 +218,9 @@ class Auth {
         this.loggedIn = true;
         this.uniqueClientSession = `${this.sessionToken}-${Math.random().toString(36).substring(6)}`;
       })
+      .catch(() => {
+        logger.warn("Failed to validate token, user moved to another breakout room");
```

Comment:
is this log correct?
it seems that this `catch` can handle other cases different of "user moved to another breakout room".

Reply:
Yeah, as far as I tested this was the only case I could get, but, if you want to, I could make it more generic, what do you think?